Slim/Formats/RemoteStream.pm

```diff
@@ -193,6 +193,8 @@ sub request {
 		# socket in a CLOSE_WAIT state and leaking.
 		$self->close();
 
+		# some 302 services omit the protocol
```

Comment:
We used to have support for other protocols like rtmp(?) - which would break this. Please make sure you only modify the `$redir` if there's no protocol defined, no matter whether it's http(s) or something else. Even if you need three lines to make it human readable 😁 .

Reply:
Yep you're right ... I'll do that tomorrow then!

Reply:
But, can you have redir (3xx) on protocols other than HTTP(S)

Reply:
You can certainly have redirects _to_ other protocols. And that's what we try to fix here.

Reply:
Undertood - see new fix proposal (still one line though :-))

Reply:
At least you left a comment in there to explain what that line does 😂 . Thanks!